src/config/firebase.js

```diff
@@ -0,0 +1,16 @@
+import firebase from 'firebase/app'
+import 'firebase/firestore'
+import 'firebase/auth'
+
+var config = {
+  apiKey: 'AIzaSyAUBUzRsIYz7MGFCWIlIoMzjl_9IMa4bho',
```

Comment:
I know :) lets decide where to store it

Reply:
yea, seems like the suggested solution is having the whole config in an `FIREBASE_CONFIG` env variable and get it with `JSON.parse(process.env.FIREBASE_CONFIG)`

Reply:
I think firebase will inject that automatically after it was set up locally, and we can make a private env variable for Travis that should work for surge.